_docs/templates/email-template.md

```diff
@@ -126,25 +185,35 @@ message it is. The header contains three parts:
 
 {% endcolumn %}
 
-{% column right:"hxCol-md-8" %}
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
 {% figure [caption:"Visual example of Sub-Header and spacing"] [class:"image bg-light border"] %}
-  ![]({{site.cdn_url}}/img/layout/email-templates/Header.svg){:width=60%}
+  ![]({{site.cdn_url}}/img/layout/email-templates/Header.svg)
```

Comment:
What's the spacing between the platform name and the message type?
<img width="212" alt="screen shot 2017-11-20 at 11 06 38 am" src="https://user-images.githubusercontent.com/1538692/33031149-fe390498-cde2-11e7-8692-27c04613aa91.png">